test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -6689,15 +6693,22 @@ def manager(self, cmd, obj_type, attrib=None, id=None, extend=None,
                     attrop = PTL_AND
 
             if oid is None:
-                return self.expect(obj_type, attrib, oid, op=op,
-                                   max_attempts=max_attempts,
-                                   attrop=attrop, offset=offset)
-            for i in oid:
-                rc = self.expect(obj_type, attrib, i, op=op,
-                                 max_attempts=max_attempts,
-                                 attrop=attrop, offset=offset)
-                if not rc:
-                    break
+                expectrc = self.expect(obj_type, attrib, oid, op=op,
+                                       max_attempts=max_attempts,
+                                       attrop=attrop, offset=offset)
+            else:
+                for i in oid:
+                    expectrc = self.expect(obj_type, attrib, i, op=op,
```

Comment:
Thanks for catching that, I did it this way because expect's dostring says:
":returns: True if attributes are as expected and False
                  otherwise"
I'll change it to try-except

Reply:
Or you can let the exception go through and just reword the doc string.  Or you can catch the exception and raise a manager exception.  I think either is better than catching the exception and returning 0.